lib/Models/Cesium.js

```diff
@@ -403,6 +403,23 @@ Cesium.prototype.computePositionOnScreen = function(position, result) {
     return SceneTransforms.wgs84ToWindowCoordinates(this.scene, position, result);
 };
 
+/**
+ * Add and remove attribution text.
+ * @param  {attribution} the attribution object of each Catalog item.
+ */
```

Comment:
Two problems with this doc:
* One jsdoc block describes one method.  So you need a separate block for `remoteAttribution` below.
* With `@param`, the first identifier in curlies is the type of the parameter, and the next identifier is the name of the parameter.  So jsdoc thinks you have a parameter named `the` of type `attribution`.

You can run `gulp docs` in the TerriaJS directory, then npm start, then navigate to `http://localhost:3002/doc` to look at the generated reference documentation.